src/Castle.Facilities.Logging/LoggerImplementation.cs

```diff
@@ -14,11 +14,14 @@
 
 namespace Castle.Facilities.Logging
 {
+	using System;
+
 	using Castle.Core.Logging;
 
 	/// <summary>
-	///   The supported <see cref = "ILogger" /> implementations
+	///   The supported <see cref = "ILoggerFactory" /> implementations.
 	/// </summary>
+	[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
```

Comment:
There are currently 131 Obsolete attributes in the code base. Can't we just delete this altogether?

Reply:
I had no intention of bumping the version to 5.0 just for this minor change, hence marking it obsolete. I agree there is a lot of stuff marked obsolete for a long time and it is time we remove much of it, I've logged an issue so we can look at doing this for Windsor 5.